Makefile.common

```diff
@@ -184,10 +181,6 @@ else
 	yamllint .
 endif
 
-# For backward-compatibility.
```

Comment:
Why removing backwards compat?

Reply:
I haven't been able to find a place where this is still in use anymore, so why keep it? 🙂